v2.0/performance-best-practices-overview.md

```diff
@@ -288,7 +288,7 @@ To avoid contention, multiple strategies can be applied:
   [normalization](https://en.wikipedia.org/wiki/Database_normalization)
   of the data to place parts of the same records that are modified by
   different transactions in different tables. Note however that this
-  is a double-edged sword, because denormalization can also increase
+  is a double-edged sword, because denormalization can also decrease
```

Comment:
The correct word was "increase" so this change is not warranted.

More details:

If a user decides to increase normalization (and thus decrease denormalization) because they want to decrease contention, they are reducing the number of copies.

Meanwhile more copies can increase performance because it parallelizes access to the data (by distributing the KV operations to them across more nodes/ranges).

So it's an unclear tradeoff, hence the word "double-edged".

Feel free to reformulate the paragraph to more clearly explain this trade-off.

Reply:
Thanks for the explanation and additional information @knz - I will probably press pause on this for now, and return to it later.